shell/common/shell_test_external_view_embedder.h

```diff
@@ -74,11 +74,13 @@ class ShellTestExternalViewEmbedder final : public ExternalViewEmbedder {
   bool SupportsDynamicThreadMerging() override;
 
   const EndFrameCallBack end_frame_call_back_;
+
```

Comment:
revert

Reply:
discussed offline. This is for consistency